src/Network/Network.Test/ScenarioTests/NetworkManagerTests.ps1

```diff
@@ -516,6 +516,191 @@ function Test-NetworkManagerSecurityAdminRuleCRUD
 	}
 }
 
+<#
+.SYNOPSIS
+Tests creating/getting/deleting new simple public networkmanager security admin Configuration/RuleCollection/Rule
+#>
+function Test-NetworkManagerSecurityAdminRuleManualAggregationCRUD
```

Comment:
consider integrating this scenario into the existing test

Reply:
it takes a lot of time to run with added delays and causes issues once you change types in NG. if it's not needed let's keep it as is